code/modules/clothing/spacesuits/rig/suits/combat.dm

```diff
@@ -172,3 +172,34 @@
 		/obj/item/rig_module/ai_container,
 		/obj/item/rig_module/actuators/combat
 		)
+
+/obj/item/weapon/rig/bunker
+	name = "bunker suit control module"
+	desc = "A powerful niche-function hardsuit utilized by Ceres' Lance to apprehend synthetics. Unstoppable in the right circumstances, and nothing more than a burden anywhere else."
+	icon_state = "bunker"
+	suit_type = "bunker suit"
+	armor = list(melee = 80, bullet = 80, laser = 80, energy = 80, bomb = 25, bio = 25, rad = 25)
```

Comment:
I thought Kyres wanted 90 resist for everything but the bomb, bio, rad?

Reply:
He told me that 80 is fine instead.

Reply:
Okay